umh-core/CHANGELOG.md

```
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Improvements
+
+### Improvements
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)